Frontend/MoptPaymentPayone/Subscribers/Payment.php

```diff
@@ -308,6 +315,13 @@ public function onShippingPaymentAction(\Enlight_Hook_HookArgs $arguments)
         if ($groupedPaymentMeans) {
             $subject->View()->sPayments = $groupedPaymentMeans;
         }
+		
+        $this->container->get('events')->notify(
+            'MoptPaymentPayone_Subscriber_Payment_paymentGrouped',
+            [
+                'payments' => $paymentMeansWithGroupedCreditcard
```

Comment:
Shouldn't this emit `$groupedPaymentMeans` as a payload and make sure it contains something useful?
Is it safe to use the same event name here?

Reply:
Yes you are right, i have updated the forked repo.
The first idea for having the same event emitted was that it will always be emitted when the payments were grouped, but by changing them to two different ones plugins can specify which one to subscribe to.